src/engine/BMInterface.php

```diff
@@ -165,8 +165,8 @@ public function load_api_game_data($playerId, $gameId, $logEntryLimit) {
                 $data['gameActionLogCount'] = $actionLogArray['nEntries'];
             }
 
-            $chatLogArray = $this->game_chat()->load_game_chat_log($game, $logEntryLimit);
-            if (empty($actionLogArray)) {
+            $chatLogArray = $this->game_chat()->load_game_chat_log($playerId, $game, $logEntryLimit);
+            if (empty($chatLogArray)) {
```

Comment:
This seemed like an obvious copy/paste-o bug, so i just fixed it.  It couldn't have ever come up yet because we don't yet support chatting any time before there are action log entries (and also `load_game_chat_log()` always returns something nontrivial anyway), but i figured i'd fix it just for cleanliness.